lib/kb_quast/baseclient.py

```diff
@@ -5,7 +5,7 @@
 #
 ############################################################
 
-from __future__ import print_function
```

Comment:
Why keep the baseclient around if the actual client is deleted?

Reply:
See above